calico_cni/policy_drivers.py

```diff
@@ -320,6 +307,21 @@ def _escape_chars(self, unescaped_string):
         return re.sub('[^a-zA-Z0-9\.\_\-]', swap_char, unescaped_string)
 
 
+class DefaultDenyInboundDriver(KubernetesAnnotationDriver):
+    """
+    This driver sets the labels and correct profile on the endpoint.
+    """
+    def apply_profile(self, endpoint):
+        profile_id = "k8s_ns.%s" % self.namespace
```

Comment:
Could you please organize this code a little more - perhaps split into steps? It's a bit ugly as a single blob of code with no comments.